conf/messages.en

```diff
@@ -197,7 +197,7 @@ commodityCode.change.hidden = Commodity code
 
 countryOfOrigin.title = Country of origin
 countryOfOrigin.h1 = Country of origin
-countryOfOrigin.p1 = Country of origin refers to the country where the goods were mostly or wholly manufactured, grown or modified. It is used for labelling purposes and affects duty rates.
+countryOfOrigin.p1 = Country of origin refers to the country where the goods were mostly or wholly manufactured, grown or modified. It's used for labelling purposes and affects duty rates.
 countryOfOrigin.p2 = If the country of origin is different to the country of departure when the goods are declared at the border, the regulations, tax and duties may differ from the guidance on this service.
```

Comment:
Small suggestion. I think we are not using "countryOfOrigin.p2" anywhere else. We can remove this. What do you think?

Reply:
yep makes sense, will leave the welsh one in for now until we get the content back for that